django/db/migrations/autodetector.py

```diff
@@ -1051,6 +1053,7 @@ def arrange_for_graph(self, changes, graph, migration_name=None):
                     migration.dependencies.append(app_leaf)
                 if i == 0 and not app_leaf:
                     new_name = "0001_%s" % migration_name if migration_name else "0001_initial"
+                    migration.initial = True
```

Comment:
I think this sets the `initial=True` for `0001_initial` while the other line above takes care of `0002_auto`.